includes/class-evf-form-task.php

```diff
@@ -200,7 +200,10 @@ public function do_task( $entry ) {
 			// Success - send email notification.
 			$this->entry_email( $this->form_fields, $entry, $form_data, $entry_id, 'entry' );
 
-			$_POST['evf_success'] = true;
+				add_filter( 'everest_forms_success', '__return_true' );
+
+				// Pass completed and formatted fields in POST.
+				$_POST['everest-forms']['complete'] = $this->form_fields;
```

Comment:
Please check why this bit of code is duplicate as the same line of code is just defined below.